pkg/webhook/common.go

```diff
@@ -44,6 +44,7 @@ const (
 	serviceAccountName = "gatekeeper-admin"
 	mutationsGroup     = "mutations.gatekeeper.sh"
 	namespaceKind      = "Namespace"
+	configResourceName = "config"
```

Comment:
https://github.com/open-policy-agent/gatekeeper/blob/c36b54c28a34f565b1159cc693cb81505dcdc390/pkg/keys/config.go#L24

Reply:
Rather than use a new `const`, I've referred to `keys.Config.Name`